### PR TITLE
Fixed issue #837  HTML element does not have lang attribute 

### PR DESCRIPTION
**Pull request by Ayesh-07**
Fixed issue #837 

## Summary

HTML element does not have Lang attribute #837 
To fix this issue, I added the `lang` attribute to the `html` element and specify the appropriate language code that reflects the primary language of the content on the page. For example, for English content, I would use `lang="en"` in the `html` tag.

## Motivation

I'm passionate about contributing to the caMicroscope organization to further my skills and knowledge. One of the issues I've successfully fixed is [Accessibility] HTML element does not have lang attribute #837. I would appreciate a review of my work to ensure it meets the project's standards. This experience has deepened my understanding of user-centered design and my ability to create intuitive user interfaces.

### CODE REVIEW

**Review by nwanduka — COMMENTED** [review]
Awesome @Ayesh-07. Let's wait for a review from our mentor.

**Review by birm — APPROVED** [review]
good simple addition!

**Pull request merged**
